src/main/flight/pid_init.c

```diff
@@ -442,8 +442,8 @@ void pidInitConfig(const pidProfile_t *pidProfile)
     pidRuntime.tpaLowMultiplier = pidProfile->tpa_low_rate / (100.0f * pidRuntime.tpaLowBreakpoint);
     pidRuntime.tpaLowAlways = pidProfile->tpa_low_always;
 
-    pidRuntime.useEzDisarm = pidProfile->ez_landing_disarm_threshold > 0;
-    pidRuntime.ezLandingDisarmThreshold = pidProfile->ez_landing_disarm_threshold * 10.0f;
+    pidRuntime.useEzDisarm = pidProfile->ez_disarm_threshold > 0;
```

Comment:
Note: we are using the boolean `useEzDisarm` only here as we could have used `pidProfile->ez_disarm_threshold > 0` as condition:

https://github.com/betaflight/betaflight/blob/8feb741e177e902e50944024bc9619eee9a52501/src/main/flight/pid.c#L992-L994

Reply:
@haslinghuis - I had thought perhaps it could be called `impact_disarm_threshold` ?

The boolean means that the comparison doesn't need to be performed each loop.  Though I am not an expert on code efficiency.  I suppose we could do three things:

- keep what we have, ie a pidRuntime boolean in pid_init.c, and use that in the function
- calculate a `const bool useEzDisarm = pidProfile->ez_disarm_threshold;` in the disarm function
- use an `if (pidProfile->ez_disarm_threshold)` test in the disarm function.

IDK which is best, or if it matters.  But if one is better than the others, I'd make a PR to make them consistent.

Reply:
![image](https://github.com/user-attachments/assets/b6029910-c85c-459a-8f42-33a9c3f9257a)

Would prefer `landing_disarm_threshold` as `impact_disarm_threshold` would indicate when drifted away and hitting obstacle before or while descending a disarm taking place ?

Reply:
Hmm. :-) tricky

I think it's intended to be for landing.  So `landing_disarm_threshold` is good for the intent.

During a position hold it isn't particularly relevant, except that if position hold is used for landing, or part of a landing process, then it is indeed a landing threshold.

So I'm favouring `landing_disarm_threshold`.

Let's see what other opinions are.

Reply:
`landing_disarm_threshold` is in line with `gps_rescue_disarm_threshold` which might be consolidated (in another PR).